src/components/utils/isDesktop.js

```diff
@@ -1,3 +1,3 @@
-const width = (window.matchMedia("(min-width: 800px)").matches)
```

Comment:
It's still used for detecting if it's a mobile device. If it's a mobile we don't want to use the camera even if it exists, because it's too taxing and does not perform well.